.travis.yml

```diff
@@ -1,11 +1,26 @@
 language: node_js
 
+sudo: required
+
+before_install:
+  - export CHROME_BIN=chromium-browser
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+
+cache:
+  directories:
+    - node_modules
+
 notifications:
-  email:
-    on_success: never
-    on_failure: change
+  email: false
 
 node_js:
-  - "stable"
+  - '7'
+  - '6'
+
+before_script:
+  - npm prune
 
-after_script: "npm install coveralls@2 && cat ./coverage/lcov.info | coveralls"
```

Comment:
no more coverage report?

Reply:
I re enable and make it works since right now the project didn't show coverage:
https://coveralls.io/github/mulesoft-labs/raml-javascript-generator

Coverage isn't high enough but at least now we measure it ;)